tests/xc_integrator.cxx

```diff
@@ -141,7 +142,123 @@ void test_xc_integrator( ExecutionSpace ex, const RuntimeEnvironment& rt,
 
 }
 
-void test_integrator(std::string reference_file, ExchCXX::Functional func, PruningScheme pruning_scheme) {
+void test_xc_integrator_uks( ExecutionSpace ex, const RuntimeEnvironment& rt,
```

Comment:
Since this function as 90% overlap with the RKS tester, I would just add the `uks` boolean to that driver and add the appropriate divergent logic to unify the implementations and reduce code duplication.